osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -41,18 +47,41 @@ public override double Calculate(Dictionary<string, double> categoryRatings = nu
             countMeh = Score.Statistics.GetOrDefault(HitResult.Meh);
             countMiss = Score.Statistics.GetOrDefault(HitResult.Miss);
 
-            // Custom multipliers for NoFail and SpunOut.
+            // guess the number minimum number of misses or slider breaks from combo
```

Comment:
Is the method in which this estimation works explained somewhere? I think it may be best split to a separate method and explained properly (or at least with a link to a doc that contains an explanation).

Reply:
There's this https://github.com/ppy/osu-performance/pull/131